packages/forma-36-react-components/src/index.ts

```diff
@@ -1,84 +1,50 @@
-export { TextLink } from './components/TextLink/TextLink';
-export { Icon } from './components/Icon/Icon';
-export { Pill } from './components/Pill/Pill';
-export { HelpText } from './components/HelpText/HelpText';
-export { FormLabel } from './components/FormLabel/FormLabel';
-export { ValidationMessage } from './components/ValidationMessage/ValidationMessage';
-export { TextInput } from './components/TextInput/TextInput';
-export { TextField } from './components/TextField/TextField';
-export { Textarea } from './components/Textarea/Textarea';
-export { CopyButton } from './components/CopyButton/CopyButton';
-export { Card } from './components/Card/Card';
-export { CardActions } from './components/Card/CardActions/CardActions';
-export * from './components/DateTime';
-export { Tooltip } from './components/Tooltip/Tooltip';
-export { EntryCard } from './components/Card/EntryCard/EntryCard';
-export { CardDragHandle } from './components/Card/CardDragHandle/CardDragHandle';
-export { IconButton } from './components/IconButton/IconButton';
-export { CheckboxField } from './components/CheckboxField/CheckboxField';
-export { Checkbox } from './components/Checkbox/Checkbox';
-export { TabFocusTrap } from './components/TabFocusTrap/TabFocusTrap';
-export { Spinner } from './components/Spinner/Spinner';
-export { Button } from './components/Button/Button';
-export { EditorToolbar } from './components/EditorToolbar/EditorToolbar/EditorToolbar';
-export { EditorToolbarButton } from './components/EditorToolbar/EditorToolbarButton/EditorToolbarButton';
-export { Dropdown } from './components/Dropdown/Dropdown';
-export { DropdownListItem } from './components/Dropdown/DropdownListItem/DropdownListItem';
-export { DropdownList } from './components/Dropdown/DropdownList/DropdownList';
-export { EditorToolbarDivider } from './components/EditorToolbar/EditorToolbarDivider/EditorToolbarDivider';
-export { SelectField } from './components/SelectField/SelectField';
-export { Select } from './components/Select/Select';
-export { Option } from './components/Select/Option/Option';
-export { InlineEntryCard } from './components/Card/InlineEntryCard/InlineEntryCard';
-export { Illustration } from './components/Illustration/Illustration';
-export { Table } from './components/Table/Table';
-export { TableBody } from './components/Table/TableBody/TableBody';
-export { TableCell } from './components/Table/TableCell/TableCell/TableCell';
-export { TableHead } from './components/Table/TableHead/TableHead';
-export { TableRow } from './components/Table/TableRow/TableRow';
-export { ToggleButton } from './components/ToggleButton/ToggleButton';
-export { AssetCard } from './components/Card/AssetCard/AssetCard';
-export { Asset, AssetIcon } from './components/Asset';
-export { Tag } from './components/Tag/Tag';
-export { Heading } from './components/Typography/Heading/Heading';
+export * from './components/Accordion';
```

Comment:
I'm fully on board with all the changes in this PR but this file. I think it's better to be specific about what you export. Otherwise, it's really easy to make a mistake and make public something that is not intended to be.

Reply:
I am on board with that 🙂 But what kind of things do you expect we want to keep private? As you know we're already seeing people importing components and types that aren't public by drilling down into absolute paths.

Reply:
I can imagine some constant being exported only for the test or even some internal component being exported for the same test purpose. It would be unfortunate for them to be exported from the main entry point.

Reply:
I think it's easier to control from the component index file, I've moved explicit exports to there. Question is if I missed anything 😅